src/features/collections/ExportPane.tsx

```diff
@@ -0,0 +1,124 @@
+import { useMemo, useState } from 'react';
+import {
+  exportSelection,
+  getSelectionTypes,
+  parseCollectionsError,
+} from '../../common/api/collectionsApi';
+import { getNarratives } from '../../common/api/searchApi';
+import { Select, Input, Button, SelectOption } from '../../common/components';
+import { uriEncodeTemplateTag as encode } from '../../common/utils/stringUtils';
+import { useSelectionId } from './collectionsSlice';
+import { useParamsForNarrativeDropdown } from './hooks';
+
+export const ExportPane = ({ collectionId }: { collectionId: string }) => {
+  const selectionId = useSelectionId(collectionId);
+  const [name, setName] = useState<string>('');
+  const [desc, setDesc] = useState<string>('');
+  const [narrativeSel, setNarrativeSel] = useState<SelectOption | undefined>();
+  const [typeSel, setTypeSel] = useState<SelectOption | undefined>();
+  const [narrativeSearch, setNarrativeSearch] = useState('');
+
+  const narrativeSearchParams = useParamsForNarrativeDropdown(narrativeSearch);
+  const narrativeQuery = getNarratives.useQuery(narrativeSearchParams);
+  const narrativeOptions = (narrativeQuery?.data?.hits || []).map((hit) => ({
+    value: [hit.access_group, hit.obj_id, hit.version].join('/'),
+    label: hit.narrative_title,
+    data: hit,
+  }));
+  const narrativeSelected = narrativeOptions.find(
+    (d) => d.value === narrativeSel?.value
+  )?.data;
+
+  const typesParams = useMemo(
+    () => ({ selection_id: selectionId ?? '' }),
+    [selectionId]
+  );
+  const typesResult = getSelectionTypes.useQuery(typesParams, {
+    skip: !selectionId,
+  });
+
+  const [triggerExport, exportResult] = exportSelection.useMutation();
+  const handleExport = () => {
+    if (!complete) return;
+    triggerExport({
+      selection_id: selectionId,
+      workspace_id: narrativeSelected?.access_group.toString(),
+      ws_type: typeSel?.value.toString(),
+      object_name: name,
+      description: desc,
+    });
+  };
+
+  const exportError = parseCollectionsError(exportResult.error);
+  const exportErrorUnknown = exportResult.error && !exportError;
+  const exportErrorText = exportErrorUnknown
+    ? 'An unknown error occurred while saving to the narrative.'
+    : exportError
+    ? `${exportError.error.httpcode}: ${exportError.error.message}`
+    : undefined;
```

Comment:
It looks like `exportError` is already known to be truthy, so this second ternary does not seem necessary. Edit: Oops that's so long as `exportErrorUnknown` is false, but the ternary covers when it is true. In that case we may want to change `undefined` to `''` as we discussed previously.